src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -88,6 +95,20 @@ public void authorConfig_noSpecialCharacter_success() throws IOException, Invali
         Assert.assertEquals(AUTHOR_CONFIG_NO_SPECIAL_CHARACTER_AUTHORS, config.getAuthorList());
     }
 
+    @Test
+    public void authorConfig_emptyLocation_success() throws ParseException, IOException {
+        RepoConfiguration expectedConfig = new RepoConfiguration(new RepoLocation(""));
+
+        AuthorConfigCsvParser authorConfigCsvParser =
+                new AuthorConfigCsvParser(AUTHOR_CONFIG_EMPTY_LOCATION_FILE);
```

Comment:
Unnecessary break, it can be together.

Our guideline
>Try to keep line length shorter than 110 characters (soft limit). But it is OK to exceed the limit slightly (hard limit: 120 chars). If the line exceeds the limit, use line wrapping at appropriate places of the line.